code/controllers/subsystem/aura.dm

```diff
@@ -91,8 +91,10 @@ SUBSYSTEM_DEF(aura)
 	var/hive_number = XENO_HIVE_NORMAL
 	///Whether we should skip the next tick. Set to false after skipping once. Won't pulse to targets or reduce duration.
 	var/suppressed = FALSE
+	///If theres anything in this list, only give our aura to specific caste types in this list
+	var/specific_castes = list()
```

Comment:
From what I can tell there is no need for the specific castes stuff since the code for fury is on puppets

Reply:
im pretty sure it could break phero icons for other castes

Reply:
Also make it var/list guh

Reply:
we dont use aura anymore i dont think